r-package/grf/vignettes/rate.Rmd

```diff
@@ -0,0 +1,216 @@
+---
+title: "Assesing heterogeneity with RATE"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{rate}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+set.seed(123)
+options(warn = -1)
+```
+
+```{r setup}
+library(grf)
+library(ggplot2)
+```
+
+This vignette gives a brief introduction to how the *Rank-Weighted Average Treatment Effect* (*RATE*) available in the function `rank_average_treatment_effect` can be used to evaluate how good estimates of conditional average treatment effects or other treatment *rules* are at distinguishing subpopulations with different treatment effects, or whether there is any notable heterogeneity present. The second part of the vignette gives a worked example from a medical setting using (synthetic) data from the SPRINT and ACCORD hypertension drug trials. For complete details we refer to the [RATE paper](https://arxiv.org/abs/2111.07966).
```

Comment:
perhaps change "treatment rules" --> "treatment prioritization rules" to be consistent with the subheader below.

Reply:
"estimates of conditional average treatment effects **or other treatment rules**" assumes that estimates of conditional average treatment effects are a kind of treatment rule but I don't know that this connection is clear/obvious until we say what we mean by a "treatment prioritization rule" in the section below. Wondering if even just switching the order here -- "[...] can be used to evaluate how good treatment prioritization rules (such as conditional average treatment effect estimates) are [...]". Alternatively, just dropping the word "other" could also do the trick, I think. Feel free to veto any of the above.

Reply:
I agree, reversing the sentence makes it nicer

Reply:
Thinking more about this, I feel like we want to be crystal clear in this first paragraph about what questions the RATE can answer. So something like 
"This vignette gives a brief introduction to how the *Rank-Weighted Average Treatment* (*RATE*) available in the function `rank_average_treatment_effect` can be used to answer questions such as,

1. Is there any significant heterogeneity in treatment effects in this sample?
2. Are my conditional average treatment effect (CATE) estimates aligned with the true CATE?
3. Does my proposed treatment prioritization rule do any better than assigning treatment uniformly at random?
4. I have two treatment prioritization rules (e.g., CATE estimators, risk estimators); is one significantly better than the other?

The second part of the vignette [...]"

I don't have strong feelings about the exact wording or ordering here (maybe @syadlowsky or @swager do) but I just want to make sure that if someone is quickly searching through available tools for evaluation they can easily and quickly - within the first paragraph - determine whether this is the right tool to use.